assets/models/system/jean-baptiste-camembert-ner/spec.yaml

```diff
@@ -1,5 +1,5 @@
 $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
-name: Jean-Baptiste-camembert-ner
+name: jean-baptiste-camembert-ner
```

Comment:
Will this automatically get published to the sample inference endpoint?

Reply:
@haritha-bendapudi Could you please confirm this?